communications/ff_msgs/srv/GetZonesMap.srv

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2017, United States Government, as represented by the
```

Comment:
This looks good but I'm wondering if we should rename the service. We already have a get zones service and I worry that it isn't clear what the difference is between the two. What do you think about GetOccupancyMap?

Reply:
I think that's a good idea. Initially I was skeptical because the 'occupancy map' name might inadvertently infer that we also have the obstacle map contents in it, but the service name should be the one clarifying the contents of the message and not the message name.